src/app/page.mdx

```diff
@@ -10,7 +10,7 @@ import links from "../data/links";
 
 ## Humanode Biomapper
 
-Generic information about what Biomapper is and how it works.
+Welcome to the Humanode Biomapper! This document will help you understand What Humanode Biomapper is and How it works
```

Comment:
I don't think it is correct to refer to the website that consists of multiple pages as a document. Typically a document is what you'd see on a single page and scroll (i.e. if it's a multi-page pdf, with multi-page as in a paper page).
Consider changing this to something else.